xgcm/grid.py

```diff
@@ -595,8 +586,26 @@ def interp(
 
         Parameters
         ----------
-        %(neighbor_binary_func.parameters.no_f)s
+        da : xarray.DataArray
```

Comment:
Wasnt this all part of another PR already? I am not quite sure why these show up as changes here.

Reply:
No I don't really understand that either :confused:

Reply:
Can you try to force resolve the merge to `theirs`? @rabernat has done that once to resolve an issue like this. https://stackoverflow.com/a/21777677

Reply:
I tried `git pull -X theirs` but that literally didn't do anything...

Then I tried rebasing this PR onto `grid_ufunc_refactor` but that hasn't got rid of the docstring changes :angry:

Reply:
It seems now at least I have all the commits I actually want to comprise this PR in a row at the end of the history... Wonder if that means I could easily make a new PR and cherry-pick the all in one go?

Reply:
Closing this in favour of #413 - cherry-picking to create a new PR did work but if anyone could explain to me what the correct thing to do to clean up the history of this PR would have been I would appreciate it!